router.go

```diff
@@ -319,6 +319,8 @@ func (p *ControllerRegister) GetContext() *beecontext.Context {
 
 // GiveBackContext put the ctx into pool so that it could be reuse
 func (p *ControllerRegister) GiveBackContext(ctx *beecontext.Context) {
+	// clear input cached data
+	ctx.Input.Clear()
```

Comment:
Actually, not only Input but also Output need to be clear. 

If fact, we need to clear whole context.